packages/protocol/contracts/libs/LibSharedConfig.sol

```diff
@@ -16,20 +16,25 @@ library LibSharedConfig {
         return
             TaikoData.Config({
                 chainId: 167,
-                maxNumBlocks: 2049, // up to 2048 pending blocks
-                blockHashHistory: 100000,
-                // This number is calculated from maxNumBlocks to make
-                // the 'the maximum value of the multiplier' close to 20.0
+                // Assuming proof time 1 hour, block time 10 seconds
+                // Level of parallelization is 60*60/10 = 360
+                // We give it extra 50% more slots, then we have 360*1.5+1=
+                maxNumBlocks: 541,
+                // Changed it to a smaller value
+                // Assuming block time 10 seconds, we support cross-chain tx
+                // to have 1 confirmation within 600 seconds (10 minutes).
+                // Then we have: 600/10 = 60.
+                blockHashHistory: 60,
```

Comment:
Are these values set to these low values for testing purposes so they get triggered faster in the testnet?

Reply:
Yes. Especially this one `blockHashHistory`.  `maxNumBlocks' should be much bigger in mainnet I believe.